languages/tree-sitter-stack-graphs-typescript/rust/lib.rs

```diff
@@ -17,9 +17,10 @@ pub mod tsconfig;
 pub mod util;
 
 /// The stacks graphs tsg path for this language.
-pub const STACK_GRAPHS_TSG_PATH: &str = "src/stack-graphs.tsg";
+pub const STACK_GRAPHS_TSG_PATH: &str = "./stack-graphs.tsg";
 /// The stack graphs tsg source for this language
-pub const STACK_GRAPHS_TSG_SOURCE: &str = include_str!("../src/stack-graphs.tsg");
+pub const STACK_GRAPHS_TSG_TS_SOURCE: &str = include_str!(concat!(env!("OUT_DIR"), "/stack-graphs-typescript.tsg"));
+pub const STACK_GRAPHS_TSG_TSX_SOURCE: &str = include_str!(concat!(env!("OUT_DIR"), "/stack-graphs-tsx.tsg"));
```

Comment:
```suggestion
const STACK_GRAPHS_TSG_TS_SOURCE: &str = include_str!(concat!(env!("OUT_DIR"), "/stack-graphs-typescript.tsg"));
const STACK_GRAPHS_TSG_TSX_SOURCE: &str = include_str!(concat!(env!("OUT_DIR"), "/stack-graphs-tsx.tsg"));
```

No need to expose the generated file paths.

Reply:
no problem but it is currently public: https://github.com/github/stack-graphs/blob/main/languages/tree-sitter-stack-graphs-typescript/rust/lib.rs#L22

Reply:
You're completely right. I thought this was exporting the path, but it's exporting the source. In that case, yes, keep it public!